evap/staff/views.py

```diff
@@ -790,12 +798,61 @@ def questionnaire_copy(request, questionnaire_id):
         else:
             return render(request, "staff_questionnaire_form.html", dict(form=form, formset=formset))
     else:
-        questionnaire = get_object_or_404(Questionnaire, id=questionnaire_id)
+        form, formset = get_identical_form_and_formset(questionnaire_id)
+        return render(request, "staff_questionnaire_form.html", dict(form=form, formset=formset))
+
+
+def find_questionnaire_name(old_questionnaire):
```

Comment:
The date was good as we change the old questionnaire name. The version number was only intended for the case that someone copies the same Questionnaire multiple times in the same day. 

So what would be the preferred course of action? Copying just adds a version number to the new one? Then the old name wouldn't change which I would prefer as well. It's a weird side effect now that I think about it.